sema4ai/vscode-client/src/viewsRobots.ts

```diff
@@ -230,30 +230,49 @@ export class RobotsTreeDataProvider implements vscode.TreeDataProvider<RobotEntr
                         }
                     );
                     if (result.success) {
-                        let actions: IActionInfo[] = result.result;
+                        let actionsAndDatasources: IActionInfo[] = result.result;
```

Comment:
Note: the method `askForActionPackageAndAction` in `actionPackage.ts` also calls `roboCommands.SEMA4AI_LIST_ACTIONS_INTERNAL`, but it doesn't care about datasources (so, with this PR it'll do the wrong thing)... Maybe we should leave `SEMA4AI_LIST_ACTIONS_INTERNAL` as is and create a `SEMA4AI_LIST_ACTIONS_AND_DATASOURCES_INTERNAL`.... or alternatively add a bool saying `collect_datasources=true|false` (default being false not to break compatibility in the other use case).